akka-docs/src/main/paradox/cluster-sharding-typed.md

```diff
@@ -0,0 +1,32 @@
+# Sharding
+
+TODO
+
+## Dependency
+
+sbt
+:   @@@vars
```

Comment:
I think there is some paradox automagics for the dependencies that will generate all three tabs nowadays. like here: https://github.com/akka/alpakka/blob/master/docs/src/main/paradox/amqp.md

Reply:
Nice, didn't know about this!